test/e2e/pkg/consumer_test.go

```diff
@@ -66,7 +83,30 @@ var _ = Describe("Consumer", Ordered, func() {
 		})
 
 		AfterAll(func() {
-			// TODO: add the consumer deletion
+			// delete the consumer
```

Comment:
if put the delete consumer in afterall, that means it always throws error when the spec is failed before creating the consumer. for example: there is an error in beforeall. I think it is better to have a spec for deleting consumer.

Reply:
Done.